src/domain/offer/boatCard/BoatCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Card from '../../../common/card/Card';
+import Grid from '../../../common/grid/Grid';
+import Section from '../../../common/section/Section';
+import LabelValuePair from '../../../common/labelValuePair/LabelValuePair';
+import { Boat } from '../utils';
+import CardHeader from '../../../common/cardHeader/CardHeader';
+import CardBody from '../../../common/cardBody/CardBody';
+import styles from './boatCard.module.scss';
+import { formatDimension, formatWeight } from '../../../common/utils/format';
+
+interface BoatCardProps {
+  boat: Boat;
+}
+
+const BoatCard: React.FC<BoatCardProps> = ({ boat }) => {
+  const { t, i18n } = useTranslation();
+  return (
+    <Card className={styles.card}>
+      <CardHeader title={t('offer.boatCard.heading')} />
+      <CardBody>
+        <Grid colsCount={2}>
+          <Section>
+            <LabelValuePair label={t('offer.boatCard.type')} value={boat.boatType} />
```

Comment:
Trivial comment, but should the localization key for this be `offer.boatCard.boatType` ?